pkg/minikube/bootstrapper/images/images.go

```diff
@@ -121,10 +121,7 @@ func etcd(v semver.Version, mirror string) string {
 
 // archTag returns a CPU architecture suffix for images
 func archTag(hasTag bool) string {
```

Comment:
lets ensure this doesnt break the old kubernetes version as seen in this part of the code
https://github.com/medyagh/minikube/blob/75cd17801287339a00efe44fbd451b4dbfadef86/pkg/minikube/bootstrapper/images/images.go#L119